_posts/2019-05-23-ion-hash-released.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: news_item
+title: "Ion Hash Specification 1.0 Released"
+date: 2019-05-23
+categories: news
+---
+This new specification defines a hash algorithm for Ion values.  It is useful when determining whether two Ion values represent the same value, or determining whether an Ion value has changed.  For example, a storage system might use Ion hashes to assert the integrity of its data.

```

Comment:
Think it'd be good to reinforce the fact that the hash algorithm works based on the Ion data model so an Ion document will produce the same hash value independent of being binary or text.